source/take_different_payments/index.html.md.erb

```diff
@@ -0,0 +1,12 @@
+---
+title: Take different kinds of payment
+weight: 110
+---
+
+# Take different kinds of payment
```

Comment:
I understand the logic of this but it feels a bit like these big features are hidden away a bit, particularly when smaller card features like blocking prepaid cards/delayed payments are in the main nav. Phone payments probably warrants it's own page (and we'll be adding more phone payment functionality at some point). Digital wallets could be a separate thing or nested somewhere.

Reply:
@miriamalice that's a fair proposal. Currently the navigation has "Take a payment" as a top level heading in the left hand nav. To distinguish from these other types of payment, perhaps should amend this to something like "Take a card payment"?  Or does content in this section also apply to the other types of payment?